src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.CodeAnalysis.Sarif.WorkItemFiling;
 using Microsoft.Json.Schema;
 using Microsoft.Json.Schema.Validation;
+using Microsoft.TeamFoundation.Build.WebApi;
```

Comment:
>using Microsoft.TeamFoundation.Build.WebApi [](start = 0, length = 43)

Pretty sure this can be removed. I remember this is only used for build specific API's and doesn't look like we are doing anything around Azure DevOps build related stuff yet.

Reply:
Good catch, removed.

---
In reply to: [376508933](https://github.com/microsoft/sarif-sdk/pull/1746#discussion_r376508933) [](ancestors = 376508933)